### ISSUE TITLE
Ensure future user can join + play game from different device/session

### ISSUE THREAD

**Issue by tednaphil** [Brain-Defrost/Brain-Defrost_FE#41]
- [x] Individual user can join from a different browser/device/session than the device game created on
- [x] Individuals can play game with individualized experience (e.g. get credit for their answer submissions)

**Comment by tednaphil** [comment]
Might need to see if updating routing settings in netlify will make way for our client-side routing! Currently getting the blank page and error seen below when trying to visit the lobby or the `joinGame` URL from an incognito window. 

![Image](https://github.com/Brain-Defrost/Brain-Defrost_FE/assets/76406423/5ac0539c-69bd-41c7-82df-2644dad2cb3f)

**Comment by EthanDuvall** [comment]
This looks like a error with our javascript not properly loading to the page ill look into it!

**Comment by tednaphil** [comment]
Tried this feature out on the development deployment.

User can access the join page - but a 403 status code is received when trying to create a new player. See below Wondering if this will even be an issue on the main deployment but wanted to make note of it when I came across it! We can talk with the BE team about it if it's an issue on the production deployment.


![Image](https://github.com/Brain-Defrost/Brain-Defrost_FE/assets/76406423/7eac3ea1-d417-4343-8731-7d6fd85c2893)
